pkg/datasource.go

```diff
@@ -132,9 +134,15 @@ func (o *OCIDatasource) QueryData(ctx context.Context, req *backend.QueryDataReq
 			o.logger.Error("error with client")
 			panic(err)
 		}
+		loggingMgmtClient, err := logging.NewLoggingManagementClientWithConfigurationProvider(configProvider)
+		if err != nil {
+			o.logger.Error("error with client")
```

Comment:
In thinking about this comment more I'm not sure this is correct. It is tied to how the result reported by the backend is processed by the frontend logic. Currently it almost appears like the frontend is expecting that it will received a 5xx error if there is a condition that prevents the backend from performing the test to be able to report that the datasource is not working.